ui/hooks/useTokensToSearch.js

```diff
@@ -17,7 +19,8 @@ import { isSwapsDefaultTokenSymbol } from '../../shared/modules/swaps.utils';
 import { toChecksumHexAddress } from '../../shared/modules/hexstring-utils';
 import { useEqualityCheck } from './useEqualityCheck';
 
-const tokenList = shuffle(
+// TODO: replace contract-metadata with list from state
+const shuffledContractMap = shuffle(
```

Comment:
Should this const live in the hook?

Reply:
The `const` is pre-existing, in this PR it is just renamed and updated the usage to be conditional based on the useTokenDetection flag. This `const` will not be used when the flag is false, in that case, won't it be better for the `const` to be outside the hook?!!

Reply:
👍